src/components/Application/ProgressBar.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Accordion, Container, Grid, Header, Label, List, Sticky } from 'semantic-ui-react'
+import { TemplateSectionPayload } from '../../utils/types'
+
+interface SectionPage {
+  sectionIndex: number
+  currentPage: number
+}
+
+interface ProgressBarProps {
+  serialNumber: string
+  templateSections: TemplateSectionPayload[]
+  sectionPage?: SectionPage
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  serialNumber,
+  templateSections,
+  sectionPage,
```

Comment:
can this be explicitly 'current' section page ?

Reply:
Sure. I'll change to that.